packages/xhr-mock/src/MockXMLHttpRequest.test.ts

```diff
@@ -3,6 +3,7 @@ import MockProgressEvent from './MockProgressEvent';
 import MockXMLHttpRequest from './MockXMLHttpRequest';
 import {MockRequest} from '.';
 import {MockError} from './MockError';
+import {isExportDeclaration} from 'typescript';
```

Comment:
This import looks unused, was it intentional?

Reply:
Probably VSCode being overzealous about auto-imports. I'll get it removed.